generate.sh

```diff
@@ -161,6 +161,16 @@ if [ $? -ne 0 ]; then
 fi
 print_log "Documentation Generated Successfully!\n"
 
+if [ -n "$APIDOCS_NAME" ] && [ -n "$APIDOCS_URL" ]; then
+  if [ "$APIDOCS_NAME" == "swagger" ]; then
+    cd ..
+    git clone -q https://github.com/swagger-api/swagger-ui.git swagger >/dev/null 2>&1
+    cd yaydoctemp/_build/html
+    cp -r ${BUILD_DIR}/../swagger/dist apidocs
```

Comment:
I don't think so. what bad effects are we talking about?
It would be an isolated directory. After cloning just move `swagger/dist/.` to `_build/html` and then remove the swagger directory. I can't think of any side effects.